docs/develop/support-ie-11.md

```diff
@@ -1,27 +1,31 @@
 ---
-title: Support Internet Explorer 11
-description: Learn how to support Internet Explorer 11 and ES5 Javascript in your add-in.
-ms.date: 05/01/2022
+title: Support Internet Explorer 11 and Microsoft Edge Legacy
```

Comment:
I think this is fine as is, but did you consider titling this page "Support older Microsoft browsers and Office versions"?

Reply:
That's a great suggestion and would definitely help with updating this article in the future. I'll make the change.